---
title: "Data_access"
author: "Ashfaq Ali"
date: "2/18/2021"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

## R Markdown


```{r libraries R}
required_pkgs = c(
  "TCGAbiolinks", 
  "GEOquery", 
  "GenomicDataCommons",
  "limma",
  "curatedTCGAData",
  "recount",
  "curatedMetagenomicData",
  "phyloseq",
  "HMP16SData",
  "caTools",
  "piano",
  "isa",
  "VennDiagram",
  "downloader",
    "gdata",
    "AnnotationDbi",
    "hgu133a.db",
  "PharmacoGx")
BiocManager::install(required_pkgs)
```

```{r, warning=FALSE, message=FALSE}
library(GEOquery)

```

## Load the Liver RNA-seq dataset GSE130970

```{r}
gse = getGEO("GSE130970")
library(SummarizedExperiment)
se = as(gse, "SummarizedExperiment")
spec("GSE130970")
```

```{r}
gset <- getGEO("GSE130970", GSEMatrix =TRUE, destdir= "../Data", AnnotGPL=FALSE)
gse <- getGEO(filename="../Data/GPL16791.soft")
se <- as(gse, "SummarizedExperiment")
```

```{r}
https://ftp.ncbi.nlm.nih.gov/geo/series/GSE130nnn/GSE130970/suppl/GSE130970_all_sample_salmon_tximport_counts_entrez_gene_ID.csv.gz
```


```{r}
library(PharmacoGx)
availablePSets(canonical = FALSE)
```

